llamafile/tinyblas_cpu_sgemm.inc

```diff
@@ -322,7 +322,8 @@ bool llamafile_sgemm(long m, long n, long k, const void *A, long lda, const void
     assert(nth > 0);
     assert(ith < nth);
 
-#if defined(__x86_64__) && QK_K == 256
+#if QK_K == 256
```

Comment:
I've always wondered, why would this ever need to be something other than 256?

Reply:
There are models where the row size is not divisible by 256. The right thing to do would have been to make it work also for such models by adding an incomplete last block. I had even started doing that, but this resulted in too many changes to the guts of `ggml`, so I abandoned it and instead added the option `QK_K = 64`. If it was up to me, I would remove support for `QK_K = 64`, but apparently there are people who still use this option.

Reply:
We can remove it in llamafile. There's always other quants to choose from for such models. For example, right now I'm working with stable diffusion and I was shocked to see the inner dimension of most tensors is an odd number!